src/components/SocialIcon.js

```diff
@@ -18,10 +19,11 @@ const StyledIcon = styled.a`
   }
 `;
 
-const SocialIcon = ({ href, icon, className }) => {
+const SocialIcon = ({ href, icon, iconName, className }) => {
   return (
     <StyledIcon href={href} className={className} target="_blank" rel="noreferrer">
       {icon}
+      <VisuallyHidden>{iconName}</VisuallyHidden>
```

Comment:
What happens if `iconName` isn't passed in? Should we add a contingency for that?